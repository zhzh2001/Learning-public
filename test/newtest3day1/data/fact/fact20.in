956785140 10