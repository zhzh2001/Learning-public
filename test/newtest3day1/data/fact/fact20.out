7.894877507e+8177183281
